.github/workflows/eks-e2e-test.yml

```diff
@@ -0,0 +1,160 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: MIT
+
+name: Reusable EKS E2E Test
```

Comment:
Instead of creating a new workflow couldn't we embed this to terraform code?

Reply:
Looking at this, we should be able to refactor it as is for ECS.